### ISSUE TITLE
Pressing OK on EditTextDialog with empty text leads to NPE

### ISSUE THREAD

**Issue by Kahtaf** [michael-rapp/AndroidMaterialPreferences#20]
Hi there, 

I'm not exactly sure when this started but fairly recent. When an EditTextDialog is opened, but there's no text entered, and OK is pressed, it leads to the following NPE. 

```
    java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.CharSequence de.mrapp.android.dialog.EditTextDialog.getText()' on a null object reference
        at de.mrapp.android.preference.EditTextPreference.onDialogClosed(EditTextPreference.java:377)
        at de.mrapp.android.preference.DialogPreference.onDismiss(DialogPreference.java:2762)
```

Looks like there needs to be a null check here because I believe null (or empty) should be an acceptable value when accepting text input.

**Comment by michael-rapp** [comment]
Thanks for reporting this issue. I was able to reproduce it. I will fix it as soon as possible.

**Comment by michael-rapp** [comment]
Should be fixed in version 5.1.5.

**Comment by Kahtaf** [comment]
Thanks for the quick turnaround! I can confirm it's working.
